Object-Containers/src/Homework/objectContainers/HobbyCountryMethod.java

```diff
@@ -0,0 +1,20 @@
+package Homework.objectContainers;
+
+import java.util.HashMap;
+
+public abstract class HobbyCountryMethod {
+
+    /**
+     *the method prints the name of the person that was given as parameter and the hobbies that
+     * the person has
+     * @param hashName the name of the HashMap where the information about the person and the hobbies are inserted
+     * @param person the name of the person object
+     */
+    public static void hobbyMethod(HashMap hashName, Person person){
```

Comment:
Un fisier nou dedicat metodei `hobbyMethod` mi se pare cam mult, metoda putea sa stea si in clasa `MainPerson`. Numele metodei nu e foarte descriptiv, putea sa se numeasca `printPersonHobbies`.